src/components/close-button/CloseButton.tsx

```diff
@@ -27,4 +34,5 @@ const CloseButton = ({ className, onClick }: CloseButtonProps) => {
     );
 };
 
-export default CloseButton;
+export { CloseButton as CloseButtonBase };
```

Comment:
usually we do this export so that we can test the component without the `injectIntl` HOC. do we need to update the import in the test?

Reply:
I removed unused export, leaving just the one with injectIntl.